src/lib/stitching/causality/schema.ts

```diff
@@ -10,7 +10,7 @@ import {
 import { readFileSync } from "fs"
 
 const blacklistedTypes: string[] = []
-const whitelistedRootFields: string[] = []
+const whitelistedRootFields: string[] = ["lot"]
```

Comment:
similar comment here, if folks thought it was a good idea I would propose moving towards preferring fields named `lotState` when referring to the causality states. If there's no interest in this change I will get over it.